opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/RuntimeEnvironment.java

```diff
@@ -125,14 +125,16 @@
     private RuntimeEnvironment() {
         configuration = new Configuration();
         configLock = new ReentrantReadWriteLock();
+        authFrameworkLock = new ReentrantReadWriteLock();
         watchDog = new WatchDogService();
         lzIndexerParallelizer = LazilyInstantiate.using(() ->
                 new IndexerParallelizer(this));
         lzSearchExecutor = LazilyInstantiate.using(() -> newSearchExecutor());
     }
 
-    /** Instance of authorization framework.*/
+    /** Instance of authorization framework and its lock.*/
```

Comment:
```suggestion
    // Instance of authorization framework and its lock.
```
If you want to have `and its lock` info here, then I would advise to remove `/**` so javadoc is not confusing.